SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -285,7 +285,7 @@ bool USpatialClassInfoManager::ShouldTrackHandoverProperties() const
 
 	const UAbstractLBStrategy* Strategy = NetDriver->LoadBalanceStrategy;
 	check(Strategy != nullptr);
-	return Strategy->RequiresHandoverData() || Settings->bEnableHandover;
+	return Strategy->RequiresHandoverData();
```

Comment:
Looks like the LayeredLBStrategy checks for handover being required by seeing if there's more than one worker. This will be true for offloading, even though there's no handover happening. Is this something we want to fix? Or are we happy with being able to override this with the CLI arg? (caveat: I checked this in a customer project which could be out of date with master)

Reply:
Meant to put this back in with the new setting - makes sense to keep it the same I think!